presto-main/src/main/java/com/facebook/presto/sql/analyzer/FeaturesConfig.java

```diff
@@ -125,6 +125,8 @@
     private int maxGroupingSets = 2048;
     private boolean legacyUnnestArrayRows;
 
+    private boolean jsonSerdeCodeGenEnabled;
```

Comment:
We don't abbreviate variable names: `jsonSerdeCodeGenerationEnabled`. Please also update the method names. We can leave the config name as is.